File: 算法/C++/598. Range Addition II.cpp
class Solution {
public:

    int maxCount(int m, int n, vector<vector<int>>& ops) {
        int min_a = m, min_b = n;
        for (auto &it : ops) {
            if (it[0] < min_a)
                min_a = it[0];
            if (it[1] < min_b)
                min_b = it[1];
        }
        return min_a * min_b;
    }
};

File: 算法/Python/598. Range Addition II.py
class Solution:
    def maxCount(self, m, n, ops):
        """
        :type m: int
        :type n: int
        :type ops: List[List[int]]
        :rtype: int
        """
        a, b = m, n
        for x, y in ops:
            a = min(a, x)
            b = min(b, y)
        return a*b
